suave/backends/redis_store_backend.go

```diff
@@ -158,44 +163,39 @@ var (
 	mempoolConfidentialStoreBid = suave.Bid{Id: mempoolConfStoreId, AllowedPeekers: []common.Address{mempoolConfStoreAddr}}
 )
 
-func (r *RedisStoreBackend) SubmitBid(bid types.Bid) error {
+func (r *RedisStoreBackend) indexBid(bid suave.Bid) error {
 	defer log.Info("bid submitted", "bid", bid, "store", r.Store)
 
-	var bidsByBlockAndProtocol []types.Bid
+	var bidsByBlockAndProtocol []suave.BidId
```

Comment:
The indexing was changed to include only the bid id and no add extra space storing the bids again.